94-appendixD.Rmd

```diff
@@ -1459,7 +1459,7 @@ The p-value's 0.05 threshold can be misleading researchers to conduct multiple b
 
 **Solution**:  
 
-The smaller $\alpha$ of 0.01 will lead to a more liberal hypothesis testing procedure, because the required p-value for reject the null hypothesis $H_0$ is smaller. 
+The greater $\alpha$ of 0.1 will lead to a more liberal hypothesis testing procedure, because the required p-value to reject the null hypothesis $H_0$ is greater. There is a higher chance that the p-value will be less than $\alpha$
```

Comment:
```suggestion
The greater $\alpha$ of 0.1 will lead to a more liberal hypothesis testing procedure, because the required p-value to reject the null hypothesis $H_0$ can be greater. There is a higher chance that the p-value will be less than $\alpha$
```